.github/workflows/release.yaml

```diff
@@ -22,7 +22,15 @@ jobs:
           registry: ghcr.io
           username: ${{ env.GH_USER }}
           password: ${{ secrets.GITHUB_TOKEN }}
-      - name: Deploy to GitHub Packages Container registry
+      - name: Deploy policy bundle to ghcr.io (for backwards compatibility)
+        run: |
+          tags=(latest ${{ env.RELEASE_VERSION}} ${{env.MINOR_VERSION }} ${{ env.MAJOR_VERSION }})
+          for tag in ${tags[@]}; do
+              oras push ghcr.io/aquasecurity/trivy-policies:${tag} \
```

Comment:
Keeping this hardcoded for trivy-policies bundle as we will also rename the repo (to trivy-checks) but want to keep the bundle for backwards compatibility.